packages/connect-thegraph-tokens/subgraph/src/aragon/mappings/MiniMeToken.ts

```diff
@@ -111,3 +102,22 @@ function _getTokenHolder(previousBlock: BigInt, miniMeTokenEntity: MiniMeTokenEn
 
   return tokenHolder
 }
+
+function _getTotalSupplyAt(tokenContract: MiniMeTokenContract, blockNumber: BigInt): BigInt {
```

Comment:
Thought it'd be nicer to process calls that could fail in separate helper functions.